charts/velero/crds/restores.yaml

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
What if the existing cluster contains the restores CRs? Can helm remove the restores CRDs if there are restored CRs within the cluster?

Reply:
We wouldn't remove existing CRs when upgrading to a higher version, is it better to keep the restores CRDs instead of removing them from the cluster for the users? users could still get restores by velero cli.

Reply:
[2]